### PR TITLE
Make configuration generic and improve docs

### PR DESCRIPTION
**Pull request by adam-hart**
Here she is!  As generic as I could find to make it, along with some updated documentation on exactly what you need and where.

@zendesk/volunteer

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +0,0 @@
-FEEDBACK_EMAIL_ADDRESS=linksf@zendesk.com
```
Comment (roseleaf) [inline .env.example]: I think .example files are helpful and should be fine, with generalized example data

File: config.js.example [hunk 2]
```diff
@@ -1,11 +0,0 @@
-export default {
```
Comment (roseleaf) [inline config.js.example]: same about .example files.. anyone have another viewpoint for me?

File: .firebaserc [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "projects": {
-    "linksf-dev": "vivid-inferno-4672",
```
Comment (thebritican) [inline .firebaserc]: linksf-dev? not vv generic...

File: .firebaserc [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-    "staging": "vivid-inferno-4672",
-    "default": "vivid-inferno-4672",
-    "production": "vivid-inferno-4672"
+    "linksf-dev": "[PROJECT_ID]",
```
Comment (roseleaf) [inline .firebaserc]: should there be any explanation about when these might be different?

**Comment by roseleaf** [comment]
overall looks awesome

**Pull request merged**